HabibSentongo/epicmail2
# #165092476 Get All Received Emails

**What does this PR do?**
View Inbox

**Description of Task to be completed?**
Using javascript fetch, a user can now view all received emails.

**How should this be manually tested?**
To test if the user can successfully view all received emails, you should be redirected to your inbox page with all of them on login.
Find the hosted app interface [here](https://habibsentongo.github.io/epicmail2/UI/)

**Important Notes**
- This project is managed is [here](https://www.pivotaltracker.com/n/projects/2319053)
- The api being used is [here](https://epicmail-sentongo-v2.herokuapp.com/)

@bekeplar @kyakusahmed @jenny @neelxie @nanfuka @Wabuluka
